### PR TITLE
stop table ACL check for query without a table name.

### PR DESCRIPTION
**Pull request by yaoshengzhe**
Vitess supports non table queries like "set flag=true"; therefore, table
ACL should be ignored for these queries.

### CODE REVIEW

**Comment by yaoshengzhe** [comment]
@guokeno0

**Comment by guokeno0** [comment]
LGTM

File: go/vt/tabletserver/query_executor.go [hunk 1]
```diff
@@ -230,6 +230,12 @@ func (qre *QueryExecutor) checkPermissions() error {
 		qre.qe.tableaclExemptCount.Add(1)
 		return nil
 	}
+
+	// empty table name, do not need a table ACL check.
+	if qre.plan.TableName == "" {
```
Comment (aaijazi) [inline go/vt/tabletserver/query_executor.go]: Seems like a good candidate for a unit test :smile:

Comment (guokeno0) [inline go/vt/tabletserver/query_executor.go]: Well, yeah, this is the most common edge case Lol
